client/components/themes-list/style.scss

```diff
@@ -2,13 +2,17 @@
 	display: flex;
 	flex-flow: row wrap;
 	list-style: none;
-	padding: 0 20px;
+	padding: 0;
 	margin: 0 -10px; // items flush against list edge
 
 	.feature-example & {
 		max-width: 1000px;
 		margin: auto;
 	}
+
+	.signup__steps & {
```

Comment:
I appreciate that there is an example directly above this, but I don't think it is [good practice](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md) to add this class here. `<ThemesList/>` is a standalone component, and it should not change its appearance depending on where it is used, especially since in this case we only want to add some space around it.

How about wrapping it in another div where you want to [add the spacing](https://github.com/Automattic/wp-calypso/blob/6905109e8d558161ae71ab084772861c1823b905/client/signup/steps/theme-selection/signup-themes-list.jsx#L56), and styling that div appropriately?